region/crateria/east.json

```diff
@@ -991,13 +991,39 @@
           "name": "West Ocean Junction (North-South Transition)",
           "nodeType": "junction",
           "nodeSubType": "junction"
+        },
+        {
+          "id": 15,
+          "name": "G-Mode Morph Junction (South West)",
+          "nodeType": "junction",
+          "nodeSubType": "junction",
+          "note": "Represents being at the Moat door with G-Mode and morph or artificial morph"
+        },
+        {
+          "id": 16,
+          "name": "G-Mode Morph Overloaded PLMs Junction (South West)",
+          "nodeType": "junction",
+          "nodeSubType": "junction",
+          "note": "Represents being at the Moat door with G-Mode, PLMs overloaded, and morph or artificial morph"
+        },
+        {
+          "id": 17,
+          "name": "G-Mode Morph Junction Direct (South West)",
```

Comment:
Indirect isn't possible here

Reply:
Junction 17 is for direct only as it requires the item to be loaded